x-pack/plugins/upgrade_assistant/README.md

```diff
@@ -66,7 +66,27 @@ To test the Elasticsearch deprecations page ([#107053](https://github.com/elasti
 
   ```
   yarn es snapshot -E path.data=./path_to_6.x_indices
-```
+  ```
+
+  **Token-based authentication**
+  Reindexing should also work using token-based authentication (implemented via [#111451](https://github.com/elastic/kibana/pull/111451)). To simulate, set the following parameters when running ES from a snapshot:
```

Comment:
I think we need another line break here, because currently this heading and paragraph are both part of the same line.

![image](https://user-images.githubusercontent.com/1238659/133315380-0b157fe1-d5d5-4c39-8c76-5e7954dd11ca.png)